doc/StandaloneHeader.md

```diff
@@ -0,0 +1,45 @@
+Standalone Header for *cupla*
+=============================
+
+cupla provides stand alone header to use the library without CMake e.g in a cling environment.
+
+
+Available Pre-compiler Definitions 
+=================================
+
+Pre-compiler definitions can be used to control the behavior of cupla/alpaka.
+The definitions must be passed via compiler instruction or defined before the accelerator header is included.
+
+- `CUPLA_STREAM_ASYNC_ENABLED` 0 == use synchronous streams, 1 == use asynchronous streams
+- `CUPLA_HEADER_ONLY` 1 == *cupla* will be used as header only library (default), else you must compile all cpp files from [src](https://github.com/ComputationalRadiationPhysics/cupla/tree/master/src)
```

Comment:
```suggestion
- `CUPLA_HEADER_ONLY`: `1` *cupla* will be used as header-only library (default), otherwise you must compile all `.cpp` files in [`src/`](https://github.com/ComputationalRadiationPhysics/cupla/tree/master/src)
```